src/cryptsetup/cryptsetup-generator.c

```diff
@@ -277,15 +277,15 @@ static int print_dependencies(FILE *f, const char* device_path) {
 static int create_disk(
                 const char *name,
                 const char *device,
-                const char *password,
+                const char *key_file,
```

Comment:
this is inherited from the crypttab files from really old distros where the column was called "password", because back then you could specify a literal password there. THis was removed eventually, already before we implemented systemd-cryptsetup but we stuck to the column name.

I changed this in the man page in 6e41f4dd916293f35d7d35cea7eed1807d7ea771. So yes it makes a ton of sense to change this in the code, too.

Maybe mention in the commit msg that this is a follow-up for 6e41f4dd916293f35d7d35cea7eed1807d7ea771 in a way